tox.ini

```diff
@@ -49,15 +49,16 @@ envlist =
     pypy3-test-core-opentracing-shim
 
     lint
-    py38-tracecontext
-    py38-{mypy,mypyinstalled}
+    tracecontext
+    mypy,mypyinstalled
     docs
     docker-tests
 
 [testenv]
 deps =
   -c dev-requirements.txt
   test: pytest
+  test: pytest-benchmark
```

Comment:
Not sure how exactly how `pytest-benchmark` works.
Does "integrates with pytest" mean that I run tests normally with `pytest` but with the addtional cli argument as defined [here](https://pytest-benchmark.readthedocs.io/en/latest/usage.html#commandline-options)?

Reply:
Yes that's exactly what it means! It just _additionally_ looks for tests that can be run as benchmarks :)